doc-content/enterprise-only/admin-and-config/exporting-importing-dashbuilder-data-proc.adoc

```diff
@@ -39,6 +44,28 @@ dashbuilder/navigation/readme.md
 VERSION
 ----
 
+
+If you want to export customized user created dashbuilder data as a ZIP file, do the following tasks:
+
+.. On the *Dashbuilder Data Transfer* page, click *Custom export*.
+.. In *Export Wizard* panel, select the datasets which you want to be present in the ZIP file and click *Next*.
+.. Select the Pages which you want to be present in the ZIP file and click *Next*.
```

Comment:
I would say it's supposed to be **pages** instead of **Pages** (given that in the previous point, the datasets has no capital "D")

Reply:
Done.